server/schedulers/hot_region.go

```diff
@@ -214,157 +218,14 @@ func (h *hotScheduler) summaryPendingInfluence() {
 				zap.Duration("zombie", maxZombieDur))
 			continue
 		}
-		if _, ok := ret[p.to]; !ok {
-			ret[p.to] = &Influence{Loads: make([]float64, len(p.origin.Loads))}
-		}
-		ret[p.to] = ret[p.to].add(&p.origin, weight)
-		if _, ok := ret[p.from]; !ok {
-			ret[p.from] = &Influence{Loads: make([]float64, len(p.origin.Loads))}
-		}
-		ret[p.from] = ret[p.from].add(&p.origin, -weight)
-	}
-	h.pendingSums = ret
-}
 
-// summaryStoresLoad Load information of all available stores.
-// it will filter the hot peer and calculate the current and future stat(rate,count) for each store
-func summaryStoresLoad(
-	stores []*core.StoreInfo,
-	storesLoads map[uint64][]float64,
-	storePendings map[uint64]*Influence,
-	storeHotPeers map[uint64][]*statistics.HotPeerStat,
-	rwTy rwType,
-	kind core.ResourceKind,
-) map[uint64]*storeLoadDetail {
-	// loadDetail stores the storeID -> hotPeers stat and its current and future stat(rate,count)
-	loadDetail := make(map[uint64]*storeLoadDetail, len(storesLoads))
-	allTiKVLoadSum := make([]float64, statistics.DimLen)
-	allTiKVCount := 0
-	allTiKVHotPeersCount := 0
-
-	for _, store := range stores {
-		id := store.GetID()
-		storeLoads, ok := storesLoads[id]
-		if !ok {
-			continue
+		if from != nil && weight > 0 {
```

Comment:
What if from == nil?

Reply:
Store is Down / Tombstone.

Reply:
Seem different from the original behavior?

Reply:
```go
		if _, ok := ret[p.to]; !ok {
			ret[p.to] = &Influence{Loads: make([]float64, len(p.origin.Loads))}
		}
		ret[p.to] = ret[p.to].add(&p.origin, weight)
		if _, ok := ret[p.from]; !ok {
			ret[p.from] = &Influence{Loads: make([]float64, len(p.origin.Loads))}
		}
		ret[p.from] = ret[p.from].add(&p.origin, -weight)
```